sycl/doc/PluginInterface.md

```diff
@@ -121,9 +121,18 @@ The trace shows the PI API calls made when using SYCL_PI_TRACE=-1.
 bound.)
 
 ### Plugin Unloading
-The plugins not chosen to be connected to should be unloaded.
-
-TBD - Unloading a bound plugin.
+The plugins not chosen to be connected to should be unloaded. When the
+environment variable SYCL_DEVICE_FILTER is set, SYCL RT will filter out plugins
```

Comment:
Probably, it would be more clear if text in PluginInterface talks only about plugin API and semantic as, in theory, plugin can be used not only by the SYCL RT.

Reply:
Thanks. Done.